src/content/docs/en/recipes/bun.mdx

```diff
@@ -2,19 +2,100 @@
 title: Use Bun with Astro
 description: Learn how to use Bun with your Astro site.
 type: recipe
-i18nReady: false
+i18nReady: true
 ---
 
-[Bun](https://bun.sh/) is a runtime for JavaScript built for speed.
+[Bun](https://bun.sh/) is an all-in-one JavaScript runtime & toolkit. See [Bun's documentation](https://bun.sh/docs) for more information.
 
-Bun recently made their first stable release. However, using Bun with Astro may reveal rough edges. Some integrations may not work as expected. Consult [Bun's official documentation for working with Astro](https://bun.sh/guides/ecosystem/astro) for details. 
+:::caution
+Bun recently made their first stable release. However, using Bun with Astro may reveal rough edges. Some integrations may not work as expected. Consult [Bun's official documentation for working with Astro](https://bun.sh/guides/ecosystem/astro) for details.
 
 If you have any problems using Bun, please [open an Issue on GitHub with Bun directly](https://github.com/oven-sh/bun/issues/new/choose).
+:::
 
-## Community Resources
+## Prerequisites
 
-Using Bun with Astro? Add your blog post or video to this page!
+- Bun installed locally on your machine. See the [installation instructions](https://bun.sh/docs/installation) in Bun's official documentation.
+
+## Create a new Astro project with Bun
+
+Create a new Astro project with Bun using the following `create-astro` command:
+
+```bash
+bunx create-astro@latest my-astro-project-using-bun
+```
+
+:::tip
+You may also [create a new Astro project from any existing Astro GitHub repository](/en/install/auto/#starter-templates) using the `--template` flag:
+```bash
+bunx create-astro@latest my-astro-project-using-bun --template eliancodes/brutal
+```
+:::
+
+## Install dependencies
+
+If you're starting a new project using `bunx create-astro`, the CLI will automatically use Bun to install dependencies and you can skip this step.
+
+Otherwise, you'll need to install your dependencies with Bun:
+
+```bash
+bun install
+```
+
+## Using Astro integrations
+
+You can also use any of the official Astro integrations with the `astro add` command:
+
+```bash
+bunx astro add react
+```
+
+## Run Astro with Bun
+
+Use the [`--bun` CLI flag](https://bun.sh/docs/cli/bunx#shebangs) to tell Bun to use it's runtime in place of Node for the `astro` command.
```

Comment:
```suggestion
:::[note]
Use the [`--bun` CLI flag](https://bun.sh/docs/cli/bunx#shebangs) before every `astro` command to use Bun's own runtime in place of Node.
:::
```
Would this be correct? Since it's before *every* command below, I feel like this call out makes sense.

Reply:
Yep! That makes sense